client/src/app/containers/Home/active_game_list.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```

Comment:
Convention would be to name this file the same as the component it contains (`ActiveGames`, or perhaps something like `GameList` would be even more descriptive?)